Snowflake.Data/Core/Session/SessionOrCreationTokens.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Snowflake.Data.Core.Session
+{
+    internal class SessionOrCreationTokens
+    {
+        private static readonly List<SessionCreationToken> s_emptySessionCreationTokenList = new List<SessionCreationToken>();
```

Comment:
This is not very clear what for is that variable. Please comment on this as discussed

Reply:
done